docs/content/commands/hugo.md

```diff
@@ -28,6 +28,7 @@ hugo
   -b, --baseURL string          hostname (and path) to the root, e.g. http://spf13.com/
   -D, --buildDrafts             include content marked as draft
   -F, --buildFuture             include content with publishdate in the future
+  -E, --buildExpired            include content already expired
```

Comment:
All content below /commands is autogenerated so no need to update these manually. Please remove these changes so this PR gets easier to read.

We usually do a ful update before a release.